DMTN-306.tex

```diff
@@ -1,79 +1,119 @@
+
+Raw image data is replicated from the US to the European facilities. Both the US and French data facilities store a complete copy of the raw image dataset. The UK facility receives the raw images corresponding to the spatial region assigned to it for processing. Data movement between these sites is facilitated by ESnet\footnote{https://es.net}, which handles transatlantic data transport; GEANT\footnote{https://geant.org}, which connects European sites; and the national research and education networks, JANET\footnote{https://www.jisc.ac.uk/janet} (UK) and RENATER\footnote{https://renater.fr} (France).
+
+The entire set of final data products, along with selected intermediate products from each campaign, is replicated from the facility where they are generated to the archive center. There, they are consolidated and incorporated into a new data release, which is delivered annually to the science collaborations for analysis \cite{10.1051_epjconf_202429501042}.
+
+The LSST Science Pipelines is the software developed by Rubin Observatory to process the survey data \cite{bosch-pipelines}. It includes advanced image processing algorithms and supporting middleware. A central component of this middleware is the Rubin Data Butler, an abstraction layer that mediates access to the data required by, or generated through, the pipelines \cite{2022SPIE12189E..11J}. The Data Butler retrieves data from persistent storage (using appropriate protocols and data formats) based on queries specified by scientifically relevant identifiers (rather than file paths), and delivers the data as in-memory Python objects to the pipelines. It also persists the in-memory objects generated by the science algorithms. Crucially, the Butler manages the location of all files within the data store, recording their locations and relationships in a relational database. Together, the file registry and the storage system where files are located constitute a \emph{repository}.
+
+Since the Rubin's data Butler is aware only of the files present at a single facility, files replicated between facilities need to be placed in the repository's data store at the location expected by the Butler. Upon reception, replicated files are ingested into the receiving facility's local Butler repository, making them available for the processing pipelines.
+
+
+\subsection{From archive to data access centers}
+\label{summit-to-data-access-centers}
+
+Annually-released data products must be distributed to approximately 15 to 20 data access centers across the America, Europe, and Asia-Pacific regions, where scientific analysis will be conducted. These distribution campaigns will be centrally coordinated by Rubin to ensure timely delivery of data releases to all analysis centers. The goal is to distribute the multi-petabyte datasets across multiple sites using the network bandwidth available at several participating sites, thereby reducing the load on the archive center \cite{RTN-086}.
```

Comment:
None in Africa or Australia?

Reply:
None in Africa. One in Australia which is included in [Asia-Pacific](https://en.wikipedia.org/wiki/Asia–Pacific).